spec/features/sessions_new.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+feature "User sign in" do
+  background do
+    User.create(email: "alice@test.com", password: "12345", full_name: "Alice")

```

Comment:
Yep. True. Suppose as it was first view test i was a little bit overwhelmed and thinking in other things. Changed it.